emotion.d.ts

```diff
@@ -0,0 +1,19 @@
+import '@emotion/react';
```

Comment:
I tend to place type files that are shared or global in a single `types` folder, inside `src`. That way, if you add any more files like this, they're collected together neatly

Reply:
Done this now. I was going to ask you about best practices around this when I did this so thanks for dropping it here